DataPipeline-Demo-Replace-Log4j-with-SLF4J-in-Maven/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.northconcepts</groupId>
+    <artifactId>Replace-Log4j-with-SLF4J-in-Maven</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>datapipeline-repo</id>
+            <url>https://northconcepts.mycloudrepo.io/public/repositories/datapipeline/</url>
```

Comment:
Can we use `maven.northconcepts.com` everywhere?

Reply:
done